packages/module-svelte/package.json

```
@@ -46,7 +46,7 @@
     "svelte": ">=5"
   },
   "dependencies": {
-    "@sveltejs/vite-plugin-svelte": "^4.0.0"
+    "@sveltejs/vite-plugin-svelte": "^5.0.0"
```

> (This bumps the lockfile to Vite v6, which could be an issue)

Oh. yeah, that's a problem. Can we use this instead? 

```suggestion
    "@sveltejs/vite-plugin-svelte": "^4.0.0 || ^5.0.0"
```

This would also not be a breaking change